internal/security/bootstrapper/command/setupacl/command.go

```diff
@@ -188,6 +204,110 @@ func (c *cmd) GetCommandName() string {
 	return CommandName
 }
 
+// createEdgeXACLTokenRoles creates secret store roles that can be used for genearting registry tokens
+// via Consul secret engine API /consul/creds/[role_name] later on for all EdgeX microservices
+func (c *cmd) createEdgeXACLTokenRoles(secretstoreToken string, rolePolicy Policy) error {
+	roleNames := c.configuration.StageGate.Registry.ACL.GetACLRoleNames()
+	if len(roleNames) == 0 {
+		c.loggingClient.Warn("found no ACL role names defined in configuration, skip create ACL roles")
+	}
+
+	for _, roleName := range roleNames {
+		// create roles based on the service keys as the role names
+		// in phase 2, we are using the same policy rule for all services
+		edgexACLTokenRole := NewRegistryRole(roleName, ClientType, []Policy{
+			rolePolicy,
+		}, false)
```

Comment:
And comment on what `false` is and why we use `false` rather than `true`

Reply:
added